modular_skyrat/modules/sec_haul/code/guns/ammo.dm

```diff
@@ -14,20 +14,20 @@
 //////////////////////
 
 /obj/item/ammo_box/advanced/b6mm
-	name = "peacekeeper ammo box (6mm)"
+	name = "6.3mm ammo box"
```

Comment:
Not that it matters because we have the ammo workbench, but these ammo box names are confusing as fuck

Reply:
"[Caliber] ammo box"